modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
But the stats will be wrong because time doesn't represent trouble, for the reason mentioned above.